packages/ai-code-completion/src/browser/ai-code-completion-preference.ts

```diff
@@ -39,6 +40,14 @@ export const AICodeCompletionPreferencesSchema: PreferenceSchema = {
                 type: 'string'
             },
             default: []
+        },
+        [PREF_AI_INLINE_COMPLETION_MAX_CONTEXT_LINES]: {
+            title: 'Maximum Context Lines',
```

Comment:
Just thinking out loud: Is that a sensible setting or would it also make sense to have two settings: Number of lines above the cursor position AND number of lines below the cursor position. That way, a user would have much more fine-grained control instead of finding a number that is then evenly distributed with +1 on the "above side" on an odd number.

Reply:
We could, but I found a more common thing is maxLines and then be to add a percentage for the prefix as a second settings (which we can do as a follow up)